TimePad/BaseTabBarController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ViewController.swift
+//  TimePad
+//
+//  Created by Emre Alpago on 10.09.2022.
+//
+
+import UIKit
+
+enum TabbarItemType {
+    case home, add, pie
+    
+    func image() -> UIImage {
```

Comment:
eğer O(1) ise function yerine computed variable kullanmak performans açısından daha verimli. var image: UIIMage { şeklinde düzeltebilirsin

https://medium.com/swift-india/functions-vs-computed-property-what-to-use-64bbe2df3916